startup.py

```diff
@@ -33,20 +41,27 @@
 parser.add_argument("--versions",         # Command line flag to print out list of all versions
                     action="store_true",  # If this flag is defined, parsedArgs.versions will be True: else False
                     help="print out a list of all McPy versions")
-parser.add_argument("--pythonlocation",  # Command line flag to define a Python interpeter location
-                    action="store",      # Stores the result of this flag in parsedArgs.pythonlocation
-                    default=None,        # If not passed, defaults to None
-                    help="location of the Python 3.8 interpeter: defaults to the one found in the path")
+parser.add_argument("--debug",          # Command line flag to set logging in DEBUG mode
```

Comment:
Should just use the same kwarg as the --versions flag does: `action="store_true"`: it'd make the code at lines 49-54 easier to read.

Reply:
Done